agent/src/main/java/com/code_intelligence/jazzer/api/BUILD.bazel

```diff
@@ -1,5 +1,14 @@
-java_library(
+java_import(
```

Comment:
We should be able to get around this `java_library`/`java_import` "hack" by exporting the `jaz.Zer` and `jaz.Ter` sources via a `filegroup` and referencing that in the sources of `java_library`.

Reply:
Ah, yes, it's much better that way! Done.